examples/resources/pingfederate_extended_properties/import.sh

```diff
@@ -0,0 +1,2 @@
+# "extendedPropertiesId" should be the ID of the extended properties to be imported
```

Comment:
Since this appears to be edit-only, this comment should follow the format of other edit only import examples, such as server settings